maketgz

```diff
@@ -168,22 +178,39 @@ echo "produce RELEASE-TOOLS.md"
 
 echo "make dist"
 targz="curl-$version.tar.gz"
-make -sj dist VERSION=$version
+make -sj dist "VERSION=$version"
 res=$?
 
 if test "$res" != 0; then
-    echo "make dist failed"
-    exit 2
+  echo "make dist failed"
+  exit 2
 fi
 
+retar() {
+  tempdir=$1
+  rm -rf "$tempdir"
+  mkdir "$tempdir"
+  cd "$tempdir"
+  gzip -dc "../$targz" | tar -xf -
+  find curl-* -type f -exec touch -c -t "$filestamp" '{}' +
```

Comment:
@vszakats was there a particular reason you selected to not set the date on the directories? They now "leak" their source datestamps into the tarball in a way the ordinary files were made not to.

Reply:
In the next line I'm only feeding the files to tar, so the directory entries are not actually included in the tarball. What you may be seeing is the automatically created directories _on extract_, which get the current timestamp.

If we include directories in `find` feeding `tar`, `tar` will include duplicate files. Dir entries will pickup all files in them, and explicit file entries are added on top of that.

Meaning, if we want tar to store directory entries and extract those with their stable timestamp, we will need a slightly different approach.

Reply:
Aaah yeah that's it of course. Silly me... thanks.